### PR TITLE
mediamarkt.com.tr

### PR DESCRIPTION
**Pull request by gork7777**
issue: #109990
Removed old rules and I added new.

### CODE REVIEW

**Review by Alex-302 — APPROVED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -119,6 +119,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+mediamarkt.*###mms-consent-portal-container
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:122]: Cannot find `#mms-consent-portal-container`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (gork7777) [reply AnnoyancesFilter/sections/cookies_specific.txt:122]: @Alex-302 You can find it in here: `www.mediamarkt.at` and also you can find `##.gdpr-cookie-layer--show` rule in `www.mediamarkt.nl`

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt:122]: So `*` is not required here, if used on one domain only.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (gork7777) [reply AnnoyancesFilter/sections/cookies_specific.txt:122]: `###mms-consent-portal-container` rule used in `mediamarkt.at` and `mediamarkt.de` sites.

**Pull request closed without merging**